src/category_theory/localization/construction.lean

```diff
@@ -29,19 +29,6 @@ in `D` (i.e. we have `hG : W.is_inverted_by G`), then there exists a unique func
 The expected property of `lift G hG` if expressed by the lemma `fac` and the
 uniqueness is expressed by `uniq`.
 
-TODO:
-1) implement a constructor for `is_localization L W` which would take
-as an input a *strict* universal property (`lift`/`fac`/`uniq`) similar to
-what is obtained here for `W.localization`. (Practically speaking, this is
-the easiest way to show that a functor is a localization.)
```

Comment:
I think this is not yet done? Is this still on your plan? Should the TODO be kept until then?

Reply:
Actually, this part was already done in a previous PR https://github.com/leanprover-community/mathlib/pull/16890

Reply:
Ah sorry. Forgot about that.